ExtensionHandler/Linux/src/tests/test_HandlerUtil.py

```diff
@@ -118,6 +121,14 @@ def test_file_fallback(self, _urllib_mock):
             # while trying to open ../net6.json
             self.assertEqual(open_mock.call_count, 2)
 
+class TestNet6FileConsistency(unittest.TestCase):
+    def test_file_consistency(self):
+        with open("net6.json", encoding="utf-8") as net6_file:
+            local = json.loads(net6_file.read())
+        remote_net6_fileurl = "https://raw.githubusercontent.com/microsoft/azure-pipelines-agent/master/src/Agent.Listener/net6.json"
```

Comment:
So this might be just me but I don't like the file-wide override on this.
Can you rather do (in front of this line)?
            # pylint: disable-next=line-too-long
That should do the job.

Reply:
@mmrazik I agree with you, the impact of `pylint: disable` should be the smallest scope possible which is in this case is the next line.
However I don't like pylint default 100 characters per line and it might be worth creating a `pylintrc` and increasing line length to something reasonable like 160.
In case we want to customize pylint even further, I've had a lot of success with this one: https://github.com/RedHatInsights/vulnerability-engine/blob/master/pylintrc

Reply:
Should I just change disable the limit for a single line (rather than the entire file) or do we want to create a `pylintrc` file?

Reply:
I would go with single line